devtools/dev_container/Dockerfile

```diff
@@ -22,3 +23,15 @@ RUN rustup component add rust-src
 RUN rustup component add llvm-tools-preview
 COPY cargo_config /root/.cargo/config
 RUN cargo install cargo-xbuild
+
+# Install fuzzing tools
+RUN cargo install afl
+RUN cargo install cargo-fuzz --version 0.10.2
```

Comment:
Nit: It would be helpful to add a comment explaining why the latest version can't be used.

Reply:
Done